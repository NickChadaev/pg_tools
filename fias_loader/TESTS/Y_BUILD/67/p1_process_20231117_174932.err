ЗАМЕЧАНИЕ:  индекс "ie4_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie3_as_adm_hierarchy" не существует, пропускается
