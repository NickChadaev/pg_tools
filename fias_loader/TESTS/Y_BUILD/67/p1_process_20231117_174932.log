2023-11-17 17:49:32 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_67 postgres /home/n.chadaev@abrr.local/Y_BUILD/67/stage_gar_c_67.csv 2023-11-13
--------------------------------------------------------
[17:49:32] ... Смоленская обл., id_region: 68, Execute: /home/n.chadaev@abrr.local/Y_BUILD/67/stage_gar_c_67.csv, process_id:25345
[17:49:32] ... Start process
[17:49:32] ...  -- Version
[17:49:32] ...  -- Очистка от данных
[17:49:32] ...  -- Убираем индексы
[17:49:32] ...  -- Set UNLOGGED
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[17:49:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_REESTR_OBJECTS_20231113_4abb5f68-421a-4c31-93ef-700bb482ba2c.XML
[17:57:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_ADDR_OBJ_20231113_4cee8d64-7332-4210-a287-50cdbd429ae0.XML
[17:58:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_HOUSES_20231113_a960e6e5-c949-41a6-ab5f-7e659b1b1903.XML
[18:04:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_STEADS_20231113_eeaf833b-0ab9-4c2b-9b5f-364d2da17cf7.XML
[18:05:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_APARTMENTS_20231113_d7121222-a712-4e47-9486-e399d6fb0bf1.XML
[18:10:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_ROOMS_20231113_51eb827e-fb3b-4fda-b8b6-2a8c4666a271.XML
[18:10:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_CARPLACES_20231113_995ee7de-cbd2-4a06-9f8c-2db9a543551b.XML
[18:10:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_ADM_HIERARCHY_20231113_3007cd32-a57b-4058-bea1-3b63be1a6557.XML
[18:19:25] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_MUN_HIERARCHY_20231113_c47df1a5-19c1-4c02-9752-b65237f8a72d.XML
[18:28:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_CHANGE_HISTORY_20231113_75918dab-a71d-4524-8372-b9bdb32010e8.XML
[18:56:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_ADDR_OBJ_DIVISION_20231113_c005c93d-3607-44dc-b211-9f1153e5c7da.XML
[18:56:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_ADDR_OBJ_PARAMS_20231113_29c339de-be10-494e-86d8-edf80d056e67.XML
[19:00:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_STEADS_PARAMS_20231113_a5699c87-62eb-49fa-a3bf-efb0bd27bba3.XML
[19:10:46] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_HOUSES_PARAMS_20231113_d734136e-9bd4-4727-929d-bb86748985bc.XML
[20:00:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_APARTMENTS_PARAMS_20231113_6d4e7c1e-c4fd-4a2c-abb8-655df3e01b49.XML
[20:07:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_ROOMS_PARAMS_20231113_526ae29c-09d1-409e-819e-1b9f5a63ee22.XML
[20:08:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_CARPLACES_PARAMS_20231113_f2aa7ba4-fb00-41a6-84c6-7eda9ff1a0f1.XML
[20:08:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/67/AS_NORMATIVE_DOCS_20231113_0b89b045-65fa-4156-a933-00926224e6ad.XML
[20:09:29] ...  -- Data finish
[20:09:29] ...  -- Set LOGGED
[20:09:49] ... Stop process
--------------------------------------------------------
