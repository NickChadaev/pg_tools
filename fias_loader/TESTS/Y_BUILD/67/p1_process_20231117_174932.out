 save_gar_version 
------------------
               82
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                82
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 687465
gar_fias.as_addr_obj: 20689
gar_fias.as_houses: 509484
gar_fias.as_steads: 60954
gar_fias.as_apartments: 353572
gar_fias.as_rooms: 647
gar_fias.as_carplaces: 0
gar_fias.as_adm_hierarchy: 712925
gar_fias.as_mun_hierarchy: 710180
gar_fias.as_change_history: 2392610
gar_fias.as_addr_obj_division: 12
gar_fias.as_addr_obj_params: 312598
gar_fias.as_steads_params: 863356
gar_fias.as_houses_params: 4105706
gar_fias.as_apartments_params: 588977
gar_fias.as_rooms_params: 697
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 119399
